tensorflow_encrypted/tensor/int100.py

```diff
@@ -307,7 +307,7 @@ def from_native(value: np.ndarray) -> 'Int100Constant':
         return Int100Constant(value, None)
 
     @staticmethod
-    def from_int100(value: Int100Tensor) -> 'Int100Constant':
+    def from_backing(value: Int100Tensor) -> 'Int100Constant':
```

Comment:
`backing` makes less sense here since it expresses what it's used for in Pond more than what it is. I suggest `from_same` instead of `from_backing`